### PR TITLE
Move links and wrap at 80 chars

### PR DESCRIPTION
**Pull request by kenyonj**
This organizes the `README` in a more editable way.

### CODE REVIEW

**Comment by kenyonj** [comment]
Love the project, @wernight ! Couldn't think of any other way to contribute other than this for now! Keep up the good work.

File: README.md [hunk 1]
```diff
@@ -1,33 +1,35 @@
 [![](https://badge.imagelayers.io/wernight/mopidy:latest.svg)](https://imagelayers.io/?images=wernight/mopidy:latest 'Get your own badge on imagelayers.io')
 
-Containerized [**Mopidy**](https://www.mopidy.com/) music server with support for [MPD clients](https://docs.mopidy.com/en/latest/clients/mpd/) and [HTTP clients](https://docs.mopidy.com/en/latest/ext/web/#ext-web).
-
+Containerized **[Mopidy]** music server with support for [MPD clients] and [HTTP
+clients].
```
Comment (wernight) [inline README.md]: You could wrap at `and [HTTP clients].` to avoid cutting `HTTP clients` in two.

**Comment by wernight** [comment]
I'm not very fond of links at the bottom in general, unless there are repeated links to same things.

If you'd like to contribute, a good thing to improve/fix would be the the ncmcpp and/or client support for visualization. See https://github.com/wernight/docker-ncmpcpp/issues/1 . Part of the issue is likely on the Mopidy side.

**Comment by kenyonj** [comment]
Feel free to go ahead and close this PR @wernight . I'll take a look at the visualization stuff in a bit, that sounds interesting.

**Pull request closed without merging**